src/frontend/e2e/tests/home.spec.ts

```diff
@@ -1,29 +1,29 @@
 import { expect, test } from "@playwright/test";
 import { getByTestID } from "../utils/selectors";
 
-//todo: populate the remaining elements
 const footer: Record<string, string> = {
   Github: "https://github.com/chanzuckerberg/czgenepi/",
   Careers:
     "https://chanzuckerberg.com/careers/career-opportunities/?initiative=science",
   "Learning Center":
     "https://help.czgenepi.org/hc/en-us/categories/6217716150804-Genomic-Epidemiology-Learning-Center",
+  //"Privacy":  "/privacy",
+  //"Terms":  "/terms",
+  //"Contact Us":  "mailto:hello@czgenepi.org"
```

Comment:
why are these commented out?

Reply:
fixed

Reply:
This is left over from PoC; these elements were failing inconsistently (failing to find) and I think during our first meeting we agreed is priority 3 so we never went back to investigate and fix. Will remove for now